ports/stm32/boards/NUCLEO_H563ZI/mpconfigboard.h

```diff
@@ -0,0 +1,80 @@
+#define MICROPY_HW_BOARD_NAME               "NUCLEO_H563ZI"
+#define MICROPY_HW_MCU_NAME                 "STM32H563ZI"
+
+#define MICROPY_PY_PYB_LEGACY               (0)
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (1)
+#define MICROPY_HW_ENABLE_RTC               (1)
+#define MICROPY_HW_ENABLE_RNG               (1)
+#define MICROPY_HW_ENABLE_ADC               (1)
+#define MICROPY_HW_ENABLE_DAC               (0) // TODO: requires DMA, which is not yet ready
+#define MICROPY_HW_ENABLE_USB               (1)
+#define MICROPY_HW_HAS_SWITCH               (1)
+#define MICROPY_HW_HAS_FLASH                (1)
+
+// The board has a 8MHz oscillator, the following gives 250MHz CPU speed
+#define MICROPY_HW_CLK_USE_BYPASS           (1)
+#define MICROPY_HW_CLK_PLLM                 (2)
+#define MICROPY_HW_CLK_PLLN                 (125)
+#define MICROPY_HW_CLK_PLLP                 (2)
+#define MICROPY_HW_CLK_PLLQ                 (2)
+#define MICROPY_HW_CLK_PLLR                 (2)
+#define MICROPY_HW_CLK_PLLVCI_LL            (LL_RCC_PLLINPUTRANGE_4_8)
+#define MICROPY_HW_CLK_PLLVCO_LL            (LL_RCC_PLLVCORANGE_WIDE)
+#define MICROPY_HW_CLK_PLLFRAC              (0)
+
+// PLL3 with Q output at 48MHz for USB
+#define MICROPY_HW_CLK_PLL3M                (25)
+#define MICROPY_HW_CLK_PLL3N                (192)
+#define MICROPY_HW_CLK_PLL3P                (2)
+#define MICROPY_HW_CLK_PLL3Q                (4)
+#define MICROPY_HW_CLK_PLL3R                (2)
+#define MICROPY_HW_CLK_PLL3FRAC             (0)
+#define MICROPY_HW_CLK_PLL3VCI_LL           (LL_RCC_PLLINPUTRANGE_1_2)
+#define MICROPY_HW_CLK_PLL3VCO_LL           (LL_RCC_PLLVCORANGE_MEDIUM)
+
+// 5 wait states, according to Table 37, Reference Manual (RM0481 Rev 1)
+#define MICROPY_HW_FLASH_LATENCY            FLASH_LATENCY_5
+
+// There is an external 32kHz oscillator
+#define MICROPY_HW_RTC_USE_LSE              (1)
+
+// UART config
+// TODO: UART1 or LPUART1? LPUART1 seems to crash system at boot...
+#define MICROPY_HW_UART1_TX                 (pin_B6) // SB14: Arduino Connector CN10-Pin14 (D1)
+#define MICROPY_HW_UART1_RX                 (pin_B7) // SB63: Arduino Connector CN10-Pin16 (D0)
+#define MICROPY_HW_UART3_TX                 (pin_D8) // SB23: ST-Link
+#define MICROPY_HW_UART3_RX                 (pin_D9) // SB18: ST-Link
+
+// Connect REPL to UART3 which is provided on ST-Link USB interface
+#define MICROPY_HW_UART_REPL                PYB_UART_3
+#define MICROPY_HW_UART_REPL_BAUD           115200
+
+// I2C buses
+#define MICROPY_HW_I2C1_SCL                 (pin_B8) // Arduino Connector CN7-Pin2 (D15)
+#define MICROPY_HW_I2C1_SDA                 (pin_B9) // Arduino Connector CN7-Pin4 (D14)
+#define MICROPY_HW_I2C2_SCL                 (pin_F1) // Connector CN9-Pin19
+#define MICROPY_HW_I2C2_SDA                 (pin_F0) // Connector CN9-Pin21
+
+// SPI buses - not yet working due to missing DMA support
```

Comment:
This should now work, see ee5e5944726983b51475ab8b687c7f5fa3671bc5

Reply:
Done, but not yet tested